content/source/docs/enterprise/private/updating-license.html.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: "enterprise2"
+page_title: "Updating License Key"
+sidebar_current: "docs-enterprise2-private-installer-rhel"
```

Comment:
This needs to be changed to something unique.

Reply:
Copypasta error, fixed!